docs/device-management/5-ota/5-legacy-releases.md

```diff
@@ -1,11 +1,17 @@
 ---
-id: release-firmware-update
-title: Releasing Firmware Updates
+id: legacy-releases
+title: Legacy Releases
```

Comment:
I know it will be described in the blog post, but I think it would be good to also include a section about migrating in this doc. We mention in another section that devices will fallback to the legacy release system when removed from cohorts, but I think it would be helpful to add a step-by-step guide describing moving a device that is currently on a release to using a deployment in a cohort. For example, if a user wanted to migrate to cohorts without having any changes take place on their fleet.